data/olm-catalog/ssp-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,458 @@
+apiVersion: operators.coreos.com/v1alpha1
```

Comment:
Similarly, this file is also not related. We probably need to add it to `.gitignore`.